2018-11-01T06:35:48.413Z - [32minfo[39m: Data Base connection success ... test
2018-11-01T06:35:48.453Z - [31merror[39m: uncaughtException: foo should be true: expected 200 to equal 401 date=Thu Nov 01 2018 10:35:48 GMT+0400 (+04), pid=11393, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=171274240, heapTotal=121278464, heapUsed=99303984, external=1540430, loadavg=[0.4794921875, 0.521484375, 0.84765625], uptime=5040, trace=[column=46, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.end, line=65, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: foo should be true: expected 200 to equal 401,     at chai.request.get.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:65:46),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-11-01T06:35:48.478Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 403 but got 200 date=Thu Nov 01 2018 10:35:48 GMT+0400 (+04), pid=11393, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=175869952, heapTotal=126521344, heapUsed=93820920, external=67096, loadavg=[0.4794921875, 0.521484375, 0.84765625], uptime=5040, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.set.end, line=78, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 403 but got 200,     at chai.request.get.set.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:78:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-11-01T06:35:48.487Z - [31merror[39m: uncaughtException: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"' date=Thu Nov 01 2018 10:35:48 GMT+0400 (+04), pid=11393, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=176140288, heapTotal=126521344, heapUsed=94526272, external=76442, loadavg=[0.4794921875, 0.521484375, 0.84765625], uptime=5040, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.end, line=110, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"',     at chai.request.get.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:110:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-11-01T06:35:48.494Z - [31merror[39m: uncaughtException: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"' date=Thu Nov 01 2018 10:35:48 GMT+0400 (+04), pid=11393, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=176140288, heapTotal=126521344, heapUsed=95005216, external=77545, loadavg=[0.4794921875, 0.521484375, 0.84765625], uptime=5040, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.set.end, line=135, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"',     at chai.request.get.set.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:135:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-11-01T06:35:48.510Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 204 but got 200 date=Thu Nov 01 2018 10:35:48 GMT+0400 (+04), pid=11393, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=176140288, heapTotal=126521344, heapUsed=97616920, external=86707, loadavg=[0.4794921875, 0.521484375, 0.84765625], uptime=5040, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.put.end, line=161, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 204 but got 200,     at chai.request.put.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:161:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-11-01T06:35:48.522Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 404 but got 200 date=Thu Nov 01 2018 10:35:48 GMT+0400 (+04), pid=11393, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=176140288, heapTotal=126521344, heapUsed=100046936, external=87213, loadavg=[0.4794921875, 0.521484375, 0.84765625], uptime=5040, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.put.send.end, line=176, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 404 but got 200,     at chai.request.put.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:176:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-11-01T06:35:48.533Z - [31merror[39m: uncaughtException: expected { status: false, platform: 'magento' } to have property 'status' of true, but got false date=Thu Nov 01 2018 10:35:48 GMT+0400 (+04), pid=11393, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=176410624, heapTotal=126521344, heapUsed=100548560, external=96413, loadavg=[0.4794921875, 0.521484375, 0.84765625], uptime=5040, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.del.send.end, line=210, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { status: false, platform: 'magento' } to have property 'status' of true, but got false,     at chai.request.del.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:210:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-11-01T06:35:48.538Z - [31merror[39m: uncaughtException: expected { status: false } to equal {} date=Thu Nov 01 2018 10:35:48 GMT+0400 (+04), pid=11393, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=176410624, heapTotal=126521344, heapUsed=100900344, external=96895, loadavg=[0.4794921875, 0.521484375, 0.84765625], uptime=5040, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.del.send.end, line=220, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { status: false } to equal {},     at chai.request.del.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:220:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-11-01T06:35:48.544Z - [31merror[39m: uncaughtException: expected { Object (code, message) } to have property 'message' of 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"' date=Thu Nov 01 2018 10:35:48 GMT+0400 (+04), pid=11393, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=176410624, heapTotal=126521344, heapUsed=101317800, external=105779, loadavg=[0.4794921875, 0.521484375, 0.84765625], uptime=5040, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.del.send.end, line=235, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (code, message) } to have property 'message' of 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"',     at chai.request.del.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:235:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
